Sources/Contentful/Link.swift

```diff
@@ -89,6 +89,25 @@ public enum Link: Codable {
             fatalError("Should not try to access sys properties on links that are resolved.")
         }
     }
+    
+    // Because sys can not be accessed in already resolved link even through it is for storing
+    // we attempt extracting sys from entity that it was resolved to.
+    // Otherwise we fall back to trying to use sys directly
+    
+    internal var persistableSys: Link.Sys {
```

Comment:
Possible to have a test for this fix?

Reply:
Sure but Ivett wanted this fix out as soon as possible so i was planning to add tests after this is released as it will take a bit of time due to RitchText complexity